instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -388,13 +394,25 @@ def _start_response(status, response_headers, *args, **kwargs):
             duration_attrs_old = otel_wsgi._parse_duration_attrs(
                 attributes, _HTTPStabilityMode.DEFAULT
             )
+
+            if wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY, None):
+                duration_attrs_old[HTTP_ROUTE] = wrapped_app_environ.get(
```

Comment:
```suggestion
                duration_attrs_old[HTTP_TARGET] = wrapped_app_environ.get(
```

to resolve the comment above. Of course this will also require updating imports.

Reply:
I would use the `spanAttributes.HTTP_TARGET` instead of the incubating symbol